packages/ui/src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -328,6 +330,13 @@ class Container extends React.Component<Props, ComponentState> {
         return newOrEditedEntity
     }
 
+    isMultiValueDisabled(entity: CLM.EntityBase, actions: CLM.ActionBase[]) {
```

Comment:
This code looks similar to the original code. I expected it to not show as green / addition if you were leaving it as it.

What's the change?

Reply:
I think it's showing as a diff (green/added) as compared to the last commit?  I didn't change anything; just put the old code back in.

Reply:
I imagine if I had squashed this commit in with the previous one, the diff would go away.  But I want(ed) to leave them separate so people can see what changes were made to address the previous comments